.circleci/config.yml

```diff
@@ -41,6 +41,10 @@ workflows:
                 root: testapp/build/outputs
                 paths:
                   - mapping/
+            - persist_to_workspace:
+                root: testapp/build/generated/source
+                paths:
+                  - buildConfig/
```

Comment:
Just a suggestion, we can add this persistence in buildConfig/config.yml file, same as apk persist..

Reply:
Yeah, that's true.. I have the same intention but I want to confirm the effect before making change in `buildConfig/config.yml`